ansible/playbooks/utils/mount_volume.yml

```diff
@@ -4,21 +4,18 @@
   hosts: all
   vars:
     example:
-      - "-e \"{'name':'/vol1', 'src':'/dev/vdc', 'fstype': 'ext3'}\""
+      - "-e \"{'VOLUME_DEVICE':'/vol1', 'VOLUME_MKFS_DEVICE':True, 'VOLUME_DEVICE_TYPE': 'ext4'}\""
 
   tasks:
 
-    - fail: msg="Must set name. Pass a value in via `-e` as JSON {{example[0]}}"
-      when: "{{ name is undefined }}"
+    - fail: msg="Must set VOLUME_DEVICE. Pass a value in via `-e` as JSON {{example[0]}}"
+      when: "{{ VOLUME_DEVICE is undefined }}"
 
-    - fail: msg="Must set src. Pass a value in via `-e` as JSON {{example[0]}}"
-      when: "{{ src is undefined }}"
+    - fail: msg="Must set VOLUME_DEVICE_TYPE. Pass a value in via `-e` as JSON {{example[0]}}"
+      when: "{{ VOLUME_DEVICE_TYPE is undefined }}"
 
-    - fail: msg="Must set fstype. Pass a value in via `-e` as JSON {{example[0]}}"
-      when: "{{ fstype is undefined }}"
 
-
-- name: Playbook to mount a volume 
+- name: Playbook to test volume before mounting
```

Comment:
What is being tested with this play? Looks like it just calls a role to mount the volume.